tests/e2e/README.md

```diff
@@ -1,37 +1,74 @@
 # WooCommerce Payments End-to-end tests
 
-E2E tests can be run locally or in GitHub Action.
+E2E tests can be run locally or in GitHub Actions. Github Actions are already configured and doesn't require any changes to run the tests.
 
-## Setup
+## Setting up & running E2E tests
 
-Setup script requires the following env variables to be configured:
+For running E2E tests locally, create a new file named `local.env` under `tests/e2e/config` folder with the following env variables (replace values as required).
+
+<details>
+<summary>Required env variables</summary>
+<p>
 
 ```
 WCP_SERVER_REPO='https://github.com/server-repo.git or git@github.com:org/server-repo.git'
 WCP_DEV_TOOLS_REPO='https://github.com/dev-tools-repo.git or git@github.com:org/dev-tools-repo.git'
 
+// Optional to see verbose output
+DEBUG=true
+```
+
+</p>
+</details>
+
+---
+
+<details>
+<summary>Choose WCPay Server instance</summary>
+<p>
+
+It is possible to use the live server or a local docker instance of WCPay server locally. On Github Actions, live server is used for tests. Add the following env variables to your `local.env` based on your preference (replace values as required).
+
+**Using Local Server on Docker**
+```
+E2E_USE_LOCAL_SERVER=true
+
 // Stripe account data. Need to support level 3 data to run tests successfully.
 E2E_WCPAY_STRIPE_TEST_PUBLIC_KEY=<stripe pk_test_xxx>
 E2E_WCPAY_STRIPE_TEST_SECRET_KEY=<stripe sk_test_xxx>
 E2E_WCPAY_STRIPE_TEST_CLIENT_ID=<stripe ca_xxx>
 E2E_WCPAY_STRIPE_TEST_WEBHOOK_SIGNATURE_KEY=<stripe whsec_xxx>
 E2E_WCPAY_STRIPE_ACCOUNT_ID=<stripe acct_id>
+```
 
-// Optional to see verbose output
-DEBUG=true
+**Using Live Server**
+
+For using a live server, you'll need to Jetpack blog token, user token & blog id from one of your test sites connected to WooCommerce Payments live account. On a connected test site, you can use the code below to extract the blog id & tokens.
+```
+Jetpack_Options::get_option( 'id' );
+Jetpack_Options::get_option( 'blog_token' );
+Jetpack_Options::get_option( 'user_tokens' );
+```
+
+Once you have the blog id & tokens, add the following ev variables to your `local.env`.
+```
```

Comment:
We'll have to set `E2E_USE_LOCAL_SERVER` as `false` for this to work, right?

Reply:
@deepakpathania I'm making changes to make local server default. So once it's ready, you need to explicitly set `E2E_USE_LOCAL_SERVER` to `false` to use the live server

Reply:
Yes, so we should mention that here in the required ENV variables list for using the live server. I just switched from local instance to live server locally and had to explicitly set it up for the clean up to work as intended.

Reply:
Made the changes :)